### PR TITLE
SOFTWARE- 2988 (Review of osg-pki-tools)

### PR DESCRIPTION
**Pull request by efajardo**
Removed the relative links and changed the usage sections to the source repo. A [pull request](https://github.com/opensciencegrid/osg-pki-tools/pull/36) was created there to move the usage docs.

### CODE REVIEW

**Comment by efajardo** [comment]
@brianhlin I made the changes.

**Comment by efajardo** [comment]
@brianhlin I added changes for [SOFTWARE-2990](https://jira.opensciencegrid.org/browse/SOFTWARE-2990) since they were on the same file and it also fixes the multiple headers for  [SOFTWARE-3090](https://jira.opensciencegrid.org/browse/SOFTWARE-3090) and the osg-ca-scripts was updated on [PR-7](https://github.com/opensciencegrid/osg-ca-scripts/pull/7).

File: docs/security/certificate-management.md [hunk 1]
```diff
@@ -690,9 +191,9 @@ Setting CA Certificates for at '$HOME/certificates'
 Setup completed successfully.
 ```
 
-#### Adding/Removing a directory of local CAs ####
+#### Adding/Removing a directory of local CAs
 
-##### Adding #####
+##### Adding
```
Comment (brianhlin) [inline docs/security/certificate-management.md]: Let's turn this section and its parent section into an h4 named `Adding a directory of local CAs`

File: docs/security/certificate-management.md [hunk 2]
```diff
@@ -733,7 +234,7 @@ include=/etc/grid-security/localca/*
 debug = 0
 ```
 
-##### Removing #####
+##### Removing
```
Comment (brianhlin) [inline docs/security/certificate-management.md]: h4 named `Removing a directory of local CAs`

File: docs/security/certificate-management.md [hunk 3]
```diff
@@ -744,9 +245,9 @@ NOTE:
     Run `osg-ca-manage refreshCA` and `osg-ca-manage fetchCRL` to commit your changes immediately.
 ```
 
-#### Removing/Adding a particular CA included in OSG CA package ####
+#### Removing/Adding a particular CA included in OSG CA package
 
-##### Removing #####
+##### Removing
```
Comment (brianhlin) [inline docs/security/certificate-management.md]: Similarly, combine these two into an h4, `Removing a CA from the OSG CA package`

File: docs/security/certificate-management.md [hunk 4]
```diff
@@ -788,7 +289,7 @@ exclude_ca = IRAN-GRID
 debug = 0
 ```
 
-##### Adding back the removed CA #####
+##### Adding back the removed CA
```
Comment (brianhlin) [inline docs/security/certificate-management.md]: h4 `Adding a CA from the OSG CA package`

File: docs/security/certificate-management.md [hunk 5]
```diff
@@ -799,7 +300,7 @@ NOTE:
     Run `osg-ca-manage refreshCA` and `osg-ca-manage fetchCRL` to commit your changes immediately.
 ```
 
-#### Inspect Installed CA Certificates ####
+#### Inspect Installed CA Certificates
```
Comment (brianhlin) [inline docs/security/certificate-management.md]: h4 `List installed CA certificates`

Only h1 and h2 sections should be title case. Can you check to make sure all the other sections have proper capitalization?

Comment (efajardo) [reply docs/security/certificate-management.md]: I have done the requessted changes @brianhlin can you please review?

Comment (efajardo) [reply docs/security/certificate-management.md]: Thanks for noticing up @brianhlin can you merge now?

**Pull request merged**